frontend/talentsearch/src/js/components/search/SearchForm.tsx

```diff
@@ -330,9 +330,10 @@ export const SearchForm: React.FunctionComponent<SearchFormProps> = ({
             idPrefix="languageAbility"
             legend="Language"
             name="languageAbility"
+            defaultSelected="any_language"
```

Comment:
I understand why you're making this change but I find it a little confusing to introduce a magic string here that doesn't exist as an enum in the schema.  Can we do better?  Maybe declare a constant in this file with a comment about what it represents?  Then use it here and also on line 124.  The logic on line 124 is ugly enough that I might suggest factoring that out into its own little function, too.  Jerbo has suggested changing the name, too.

Reply:
If you do refactor the logic on line 124, please use the real constants, like `LanguageAbility.English` instead of `"ENGLISH"`.